packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -179,6 +191,35 @@ class DDSCarousel extends HostListenerMixin(LitElement) {
     this.start = Math.max(start - pageSize, 0);
   }
 
+  /**
+   * Handles `touchstart` event.
+   */
+  private _handleTouchStartEvent(event: TouchEvent) {
+    this._startPos = event.touches[0].clientX;
+    this._startTime = new Date().getTime();
+  }
+
+  /**
+   * Handles `touchend` event.
+   */
+  private _handleTouchEndEvent(event: TouchEvent) {
+    const { _startPos, _startTime } = this;
+    const { pageSize, start, _total: total } = this;
+    const allowedTime = 300; // max time allowed to do swipe
+    const threshold = 75; // min distance traveled to be considered swipe
```

Comment:
Should put these constants in private class fields or out-of-the class variables.

Reply:
Done =)